README.md

```diff
@@ -118,6 +118,7 @@ Keep-Alive: timeout=5
 | environment               | `UNLEASH_ENVIRONMENT`            | `undefined`     |    no    | If set this will be the `environment` used by the proxy in the Unleash Context. It will not be possible for proxy SDKs to override the environment if set.                                                                                                                                                                                                                                                                                                               |
 | projectName               | `UNLEASH_PROJECT_NAME`           | `undefined`     |    no    | The projectName (id) to fetch feature toggles for. The proxy will only return know about feature toggles that belongs to the project, if specified.                                                                                                                                                                                                                                                                                                                      | |
 | logger                    | n/a                              | SimpleLogger    |    no    | Register a custom logger.                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| useJsonLogger             | `JSON_LOGGER`                    | false    |    no    | Use the one-line JSON logger.                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
I think this reflects the new behavior better, WDYT?
```suggestion
| useJsonLogger             | `JSON_LOGGER`                    | false           |    no    | Sets the default logger to log as one-line JSON. This has no effect if a custom logger is provided.                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Reply:
Yep, that makes a lot more sense!